chatgpt-shell.el

```diff
@@ -1476,11 +1470,12 @@ With prefix REVIEW prompt before sending to ChatGPT."
   (interactive "P")
   (unless (region-active-p)
     (user-error "No region active"))
-  (let ((chatgpt-shell-prompt-query-response-style 'shell))
+  (let* ((chatgpt-shell-prompt-query-response-style 'shell)
```

Comment:
We can use `let` instead of `let*` in this case.

Reply:
good point. will address

Reply:
oh sorry, that one need let* cause two local variable used there.

Reply:
> oh sorry, that one need let* cause two local variable used there.

I'm only seeing `chatgpt-shell-prompt-query-response-style` and `region-text` declared and they don't seem to depend on each other. Why use `let*` instead of `let`?

Reply:
i see. i thought let only allow one assignment instead multiple. Good to know this. i'll use let.